app/aggregates/reviewing/available_reviewer_actions.rb

```diff
@@ -46,10 +62,6 @@ def non_means?
       work_stream == Types::WorkStreamType['non_means_tested']
     end
 
-    def funding_decision_actions_enabled?
-      FeatureFlags.adding_decisions.enabled? && has_decisions
```

Comment:
`has_decisions` in the `initialize` and `for` methods can go as well. I think this is the only place we were using it.